portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadData() {
+    const commentsContainer = document.getElementById('comments-container');
```

Comment:
nit: indentation should be two spaces, not four

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Reply:
I've installed eslint to fix the style automatically